File: daily/20180121/example_gomock/hello/hello.go
package hello

// HelloWorld :
func HelloWorld(h Hello, needCall bool) {
	if needCall {
		h.Hello("hello world")
	}
}

// Hello :
type Hello interface {
	Hello(s string)
}

type hello struct {
	fn func(string)
}

func (h *hello) Hello(s string) {
	h.fn(s)
}

File: daily/20241117/example_deno/hello.mjs
import {parseArgs} from "./with-help.dist.js";

const args = parseArgs(Deno.args, {
    description: "試しにbundleされたwith-helpを使ってみる",
    string: ["name"],
    required: ["name"],
    default: {name: "world"},
});

console.log(`Hello, ${args.name}!`);